frontend/packages/dev-console/src/utils/__tests__/pipeline-test-data.ts

```diff
@@ -272,79 +272,8 @@ export const constructPipelineData = {
     specificPipelineData.pipelineRuns[DataState.CANCELLED3],
   ],
 };
-
 export const mockRunDurationTest = [
-  {
-    apiVersion: 'tekton.dev/v1alpha1',
-    kind: 'PipelineRun',
-    metadata: {
-      name: 'simple-pipeline-aeocih',
-      namespace: 'tekton-pipelines',
-    },
-    spec: {
-      pipelineRef: {
-        name: 'simple-pipeline',
-      },
-    },
-    status: {
-      completionTime: '2019-10-29T11:57:53Z',
-      conditions: [
-        {
-          lastTransitionTime: '2019-10-29T11:57:53Z',
-          reason: 'Succeeded',
-          status: 'True',
-          type: 'Succeeded',
-        },
-      ],
-    },
-  },
-  {
-    apiVersion: 'tekton.dev/v1alpha1',
-    kind: 'PipelineRun',
-    metadata: {
-      name: 'simple-pipeline-aeocih',
-      namespace: 'tekton-pipelines',
-    },
-    spec: {
-      pipelineRef: {
-        name: 'simple-pipeline',
-      },
-    },
-    status: {
-      conditions: [
-        {
-          lastTransitionTime: '2019-10-29T11:57:53Z',
-          reason: 'Succeeded',
-          status: 'True',
-          type: 'Succeeded',
-        },
-      ],
-      startTime: '2019-10-29T11:56:40Z',
-    },
-  },
-  {
-    apiVersion: 'tekton.dev/v1alpha1',
-    kind: 'PipelineRun',
-    metadata: {
-      name: 'simple-pipeline-aeocih',
-      namespace: 'tekton-pipelines',
-    },
-    spec: {
-      pipelineRef: {
-        name: 'simple-pipeline',
-      },
-    },
-    status: {
-      completionTime: '2019-10-29T11:57:53Z',
-      conditions: [
-        {
-          lastTransitionTime: '2019-10-29T11:57:53Z',
-          reason: 'Succeeded',
-          status: 'True',
-          type: 'Succeeded',
-        },
-      ],
-      startTime: '2019-10-29T11:56:40Z',
-    },
-  },
+  pipelineTestData[PipelineExampleNames.SIMPLE_PIPELINE].pipelineRuns[DataState.SUCCESS],
+  pipelineTestData[PipelineExampleNames.WORKSPACE_PIPELINE].pipelineRuns[DataState.SUCCESS],
+  pipelineTestData[PipelineExampleNames.CLUSTER_PIPELINE].pipelineRuns[DataState.SUCCESS],
```

Comment:
Hard approve of this change. Very nicely done, consolidating test data is awesome!